qa/workunits/suites/fsstress.sh

```
@@ -1,5 +1,22 @@
 #!/bin/bash
 
+if [ ! -f /usr/lib/ltp/testcases/bin/fsstress ]
+then
+    path=`pwd`
+    mkdir -p /tmp/fsstress
+    cd /tmp/fsstress
+    wget -q -O /tmp/fsstress/ltp-full.tgz http://ceph.com/qa/ltp-full-20091231.tgz
+    tar xzf /tmp/fsstress/ltp-full.tgz
+    rm /tmp/fsstress/ltp-full.tgz
+    cd /tmp/fsstress/ltp-full-20091231/testcases/kernel/fs/fsstress
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)